### DISCUSSION TITLE
Convert Tailwind 4px spacing to REM

### DISCUSSION THREAD

**Discussion by mrleblanc101** [tailwindlabs/tailwindcss#11501] [Ideas]
Currently I have a `w-9` which I want to convert to `min-w-9`.
In v3, even with JIT, tailwind will not generate this unit.
I have 2 choice then, extend the `tailwind.config.js`, or use dynamic value.
For dynamic value, I can do `min-w-[2.25rem]` (9 divided by 4) but it's annoying have to do the math everytime.
Wouldn't it be nice if we used "untiless" dynamic value, that tailwind would convert to rem automatically so that `min-w-[9]` become `min-width: 2.25rem` ?

**Comment by mrleblanc101** [comment]
One edge case I see is for line-height, we wouldn't want to convert `leading-[1.2]` to rem as unitless value are perfectly normal for `line-height`. Not sure if there is any other property that support this tho.

**Comment by mybearworld** [comment]
You can use `min-w-[theme(spacing.9)]`. This will generate the following CSS:
```css
.min-w-\[theme\(spacing\.9\)\] {
  min-width: 2.25rem;
}
```
Play: https://play.tailwindcss.com/gCE9bqVZtX

**Reply by mrleblanc101** [reply]
WoW thank you, this is exactly what I needed. Now I just need tailwind@4 to support spacing arbitrary value when using JIT mode. Since `spacing.43` would not work since it's not in the tailwind.config.js by default.

**Reply by mybearworld** [reply]
There's probably a better way to do things than to use a 43 on the spacing scale, as the spacings there are are specifically made so that they fit well together, but if you really need it, it's possible with `min-w-[calc(theme(spacing.1)*43)]`. [Play](https://play.tailwindcss.com/3S85NbmqTU)

**Reply by mrleblanc101** [reply]
Indeed, but there are many many reason why the spacing scale is not good enough useless you add your own value. It make sense for font-size, but once you go in big number, the value become too far appart.

But as I said, I already saw an accepted proposal for arbitrary value when using JIT mode for Tailwind 4.
